test/pj-rehearse-integration/candidate/ci-operator/jobs/super/duper/super-duper-master-presubmits.yaml

```diff
@@ -157,50 +169,6 @@ presubmits:
             cpu: 10m
       serviceAccountName: ci-operator
     trigger: ((?m)^/test( all| integration| changed),?(\s+|$))
-  - agent: kubernetes
```

Comment:
Why did this change?

Reply:
it changed order. No worries, I added only the corresponding changes. I will update the integration tests in another PR.